### PR TITLE
complete login with email and pass

### PR DESCRIPTION
**Pull request by crashreact**

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
     <h1>Login work stated</h1>
+    <h2>Initial login with email and password done</h2>
```
Comment (pktherock) [inline login.html]: write the proper function name

**Review by pktherock — CHANGES_REQUESTED** [review]

File: login.html (hunk 1 above)
Comment (crashreact) [reply login.html]: ok got it. i am working on it

Comment (crashreact) [reply login.html]: function name corrected please review this

**Review by pktherock — APPROVED** [review]

File: login.html (hunk 1 above)
Comment (pktherock) [reply login.html]: ok you are not good to go

**Pull request merged**